src/queue.js

```diff
@@ -212,4 +215,16 @@ Queue.prototype._dequeuePendingRequest = function(item) {
   }
 };
 
+Queue.prototype._maybeLog = function(item) {
+  if (this.logger && this.options.verbose) {
+    var message = _.get(item, 'data.body.trace.exception.message');
```

Comment:
Should this also handle logging messages which are not part of a trace/trace_chain?